stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinterRegistry.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.network;
+
+public interface AsyncPrettyPrinterRegistry {
+  public boolean register (String headerName, AsyncPrettyPrinterFactory factory);
```

Comment:
nit: no space after `register` and before `(` (and same for the next line with the `lookup` method)